jitbuilder/CMakeLists.txt

```diff
@@ -61,7 +62,7 @@ endif()
 create_omr_compiler_library(
 	NAME    jitbuilder
 	OBJECTS ${JITBUILDER_OBJECTS}
-	DEFINES PROD_WITH_ASSUMES
+	DEFINES PROD_WITH_ASSUMES JITTEST JITBUILDER_SPECIFIC
```

Comment:
`JITBUILDER_SPECIFIC` isn't used anywhere; It is referenced in the Makefiles, but that's incorrect, and should be removed. 

Hypothetically `JITBUILDER_PROJECT_SPECIFC`would be the correct macro, but I don't think it's used anywhere at the moment.

Reply:
Definitely we should open an issue about getting rid of JITTEST, but I think this is the right choice right now.

Reply:
I removed JITBUILDER_SPECIFIC.